src/neural/CL/cl2.hpp

```diff
@@ -86,39 +86,39 @@
  * fixes in the new header as well as additional OpenCL 2.0 features.
```

Comment:
IIRC last time this file was autoformatted it broke the OpenCL backend.  Suggest reverting.